source/extensions/transport_sockets/tls/ssl_socket.cc

```diff
@@ -221,15 +223,19 @@ void SslSocket::drainErrorQueue() {
     saw_error = true;
 
     if (failure_reason_.empty()) {
-      failure_reason_ = "TLS error:";
+      failure_reason_ = failure_reason_ = new_ssl_failure_format ? "TLS_error:" : "TLS error:";
```

Comment:
Emm...is it a duplicated assignment? `failure_reason_ = failure_reason_ = ...'

Reply:
yea, what I suggested is `failure_reason_ = new_ssl_failure_format ? "TLS_error:" : "TLS error:"`
@botengyao , typo?

Reply:
oops, thanks!